coopterrones/color-palette
# Setup/files

#### What's this PR do?
- Setup file architecture

#### Where should the reviewer start?
- check file tree 

#### How should this be manually tested?
- check for typos
- make sure the file structure and components are adequate for the project
-
